File: 3533-snake-in-matrix/3533-snake-in-matrix.cpp
class Solution {
public:
    int finalPositionOfSnake(int n, vector<string>& commands) {
        int row = 0;
        int col = 0;

        for(string s : commands){
            
            if (s=="DOWN")
                col++;        
            else if(s=="RIGHT")
                row++;
            else if (s=="LEFT")
                row--;
            else if (s=="UP")
                col--;
        }
        return (col*n)+row;
    }
};

File: 3533-snake-in-matrix/3533-snake-in-matrix.py
class Solution:
    def finalPositionOfSnake(self, n: int, commands: List[str]) -> int:
        row = 0
        col = 0

        for i in commands:

            if i == "DOWN":
                col +=1        
            elif i == "RIGHT":
                row+=1
            elif i == "LEFT":
                row-=1
            elif i == "UP":
                col-=1
        
        return (col*n)+row